Formula/got-portable.rb

```diff
@@ -0,0 +1,38 @@
+class GotPortable < Formula
```

Comment:
Just call this `got`. Everyone else seems to: https://repology.org/project/got-game-of-trees/versions